embedded-graphics/src/geometry/point.rs

```diff
@@ -279,6 +299,41 @@ where
 mod tests {
     use super::*;
 
+    #[test]
+    fn convert_positive_to_u32_tuple() {
+        let p = Point::new(10, 20);
+
+        let tuple: (u32, u32) = p.try_into().unwrap();
+
+        assert_eq!(tuple, (10u32, 20u32));
+    }
+
+    #[test]
+    fn convert_i32_max_to_u32_tuple() {
+        let p = Point::new(i32::max_value(), i32::max_value());
+
+        let tuple: (u32, u32) = p.try_into().unwrap();
+
+        // Literal value taken from [here](https://doc.rust-lang.org/std/primitive.i32.html#method.max_value)
+        assert_eq!(tuple, (2147483647, 2147483647));
+    }
+
+    #[test]
+    #[should_panic]
```

Comment:
This test and the next one shouldn't use `unwrap` together with the `should_panic` attribute, because this way the tests would also pass if, for example, `Point::new` panics. Explicitly testing the return value of `try_into` would remove this possibility for false positives.

Reply:
I first tried an `assert_eq!(tuple, Err(core::num::TryFromIntError))` but `TryFromIntError` can't be constructed. I went with `assert!(tuple.is_err())` instead. Is there a better check?

Reply:
> Is there a better check?

IMO `assert!(tuple.is_err())` is a good check in this case. There is only one possible error type which doesn't have any parameters, so that both asserts in your comment are equivalent.

Reply:
Sweet 👌